src/interface.jl

```diff
@@ -139,6 +139,16 @@ Add a new edge `e` to `g`. Return false if add fails
 """
 add_edge!(x, e) = _NI("add_edge!")
 
+
+"""
+add_edges!(g, es)
+
+Add a new edges `e in es` to `g`. Return true if any new edges are added.
+Implicitly adds new vertices needed.
+"""
+add_edges!(x, es) = _NI("add_edges!")
+
+
```

Comment:
If we make this part of the interface, then we should have a default behavior for it: `all([add_edge!(g, e) for e in es])`, I think.